src/main/kotlin/dev/ocpd/jsensible/internal/java/ParameterIsNullable.kt

```diff
@@ -0,0 +1,28 @@
+package dev.ocpd.jsensible.internal.java
+
+import com.tngtech.archunit.core.domain.JavaMethod
+import com.tngtech.archunit.lang.ArchCondition
+import com.tngtech.archunit.lang.ConditionEvents
+import com.tngtech.archunit.lang.SimpleConditionEvent.violated
+
+object ParameterIsNullable {
+
+    /**
+     * All parameters within the method should be annotated with @Nullable
+     */
+    fun parameterIsNullable(): ArchCondition<JavaMethod> =
```

Comment:
Can we do this with `DescribedPredicate<JavaMethod>`?

Reply:
I have tried, but it didn't go well.